lib/Form/Processor/ContentFormProcessor.php

```diff
@@ -117,6 +117,10 @@ public function processPublish(FormActionEvent $event)
         $draft = $this->saveDraft($data, $form->getConfig()->getOption('languageCode'));
         $content = $this->contentService->publishVersion($draft->versionInfo);
 
+        $event->setPayload('contentDraft', $draft);
```

Comment:
I'd name it `content` and do not pass `versionInfo` and `contentInfo` because they can be obtained from `content` directly.